siac/walletcmd.go

```diff
@@ -224,8 +227,11 @@ func walletloadseedcmd() {
 }
 
 // walletloadsiagcmd loads a siag key set into the wallet.
-func walletloadsiagcmd(keyfiles string) {
-	password, err := speakeasy.Ask("Wallet password: ")
+func walletloadsiagcmd() {
+	fmt.Print("Enter Siag Location: ")
```

Comment:
This prompt doesn't make it clear that the user is supposed to specify multiple files.